graphql_api/types/comparison/comparison.py

```diff
@@ -45,6 +46,25 @@ def resolve_impacted_files(
     return command.fetch_impacted_files(comparison_report, comparison, filters)
 
 
+@comparison_bindable.field("impactedFilesNew")
+@convert_kwargs_to_snake_case
+@sync_to_async
+def resolve_impacted_files_new(
+    comparison_report: ComparisonReport, info, filters=None
+) -> List[ImpactedFile]:
+    command: CompareCommands = info.context["executor"].get_command("compare")
+    comparison: Comparison = info.context.get("comparison", None)
+
+    if filters:
```

Comment:
I'd also add `if filters and comparison` cause there's some scenarios where the comparison isn't loaded into the context